packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -371,7 +388,11 @@ async function launchDesktopApp(app: OfficeApp, manifest: ManifestInfo, document
       );
     }
     await launchApp(app, await getOutlookExePath());
-  } else {
+  } else if (manifestPath.endsWith(".json"))
+  {
+    await launchApp(app, await getWXPExePath(app));
+  } else
+  {
```

Comment:
convention is that this should be on the previous like with the else statement (like it was before your change)